tests/json_schema_test_suite_test.rs

```diff
@@ -66,19 +66,22 @@ fn translation_typecheck_test(
     test_case: TestCase,
 ) {
     let contract = if test_case.schema.is_object() {
-        root_schema(&dbg!(serde_json::from_value(test_case.schema).unwrap()))
+        convert(
+            &dbg!(serde_json::from_value(test_case.schema).unwrap()),
+            None,
+        )
     } else {
         let schema: Schema = dbg!(serde_json::from_value(test_case.schema).unwrap());
 
-        wrap_contract(
+        wrap_inline_lib(
             Environment::empty(),
             schema.as_predicate(&mut Default::default()).into(),
         )
     };
 
     let instance: RichTerm = serde_json::from_value(test_case.instance).unwrap();
 
-    // FIXME: this relies on `./lib/predicates.nix` being accessible from the
+    // FIXME: this relies on `./lib/predicates.ncl` being accessible from the
     // working directory
```

Comment:
Haven't we fixed that a long time ago with the predicates inlining?

Reply:
Oops, I forgot to answer that. I absolutely don't know, I just encountered this by accident and fixed the file extension. But I think you're right, there's no reason the tests should need that anymore.